src/gtfs/mod.rs

```diff
@@ -355,21 +351,106 @@ pub fn read_from_zip<P: AsRef<Path>>(p: P, configuration: Configuration) -> Resu
 /// This method makes it possible to read from a variety of sources like read a GTFS
 /// from the network.
 ///
-/// ```
-// let url = "http://some_url/gtfs.zip";
-// let resp = reqwest::blocking::get(url)?; // or async call
-// let data = std::io::Cursor::new(resp.bytes()?.to_vec());
-// let model = transit_model::gtfs::from_read(data, &url, configuration)?;
+/// ```ignore
```

Comment:
Why do we need to ignore it? Because `reqwest` is not part of the dependency-tree? Why didn't it fail before?

For this specific documentation example, we might be able to find another example that doesn't require to bring a big dependency?

Reply:
yes reqwest is not a dependency, it cannot be build.
Before the was only 2 `/` so it was not build at all :sweat_smile: 
Do you have an example in mind? It's true that it's a shame not to build all examples, but I think it can be nice to have a real use case here.

Reply:
Opened a PR targeting this branch to explore activation of doc test: https://github.com/CanalTP/transit_model/pull/782
Let you take only what's fine out of it.

Reply:
Bringing `reqwest` will considerably increase the build time as it pull an important number of transitive dependencies. This is one of the reason `reqwest` was removed from `transit_model` in the first place. So even if what you propose does work, I'm not sure we want to go that way.

Also, it does bring a dependency to the network inside the tests, I'm not entirely sure if it's risky or not but usually, a test that does not depend on _the outside_ is more reliable.

Reply:
OK, locally I didn't hit the `reqwest` build time (but it looks like CI doesn't have it so easy).
Added to network, and to the files that would move...

Should we add only the transit_model:: for Errors? (and the complete output line for ntfs doc-test)?